WcaOnRails/app/views/registrations/psych_sheet_event.html.erb

```diff
@@ -25,24 +25,38 @@
     </thead>
 
     <tbody>
-      <% @registrations.each do |registration| %>
+      <% current_pos = 0 %>
```

Comment:
Thanks @larspetrus, I knew that was going to be the answer :) I'll move it back into the controller.

This code loops through the registrations and puts a ranking number ( if 2 people have the same time they get the same number ) against them for this event in the competition. Then the view loops through them in the same manner and prints it out. As the controller and the view were doing basically the same thing twice I thought I'd try just doing it once in the view. Shouldn't be a big deal to move back though to make it easier to maintain and test.